less/type.less

```diff
@@ -238,6 +238,67 @@ ol {
   }
 }
 
+ul.tw-list-styled {
+  padding: 0;
+  margin: 32px 0;
```

Comment:
Can we avoid using margin-top?  Generally speaking we only use margin-bottom, so top spacing is determined by the padding of the container or the element it follows, if we need to overrule this in a specific case, we have the m-t-3, p-t-2 helper classes.